packages/upload/test/adding-files.test.js

```diff
@@ -190,6 +190,13 @@ describe('adding files', () => {
       expect(upload.files.length).to.equal(1);
     });
 
+    it('should allow files with extensions containing multiple dots', () => {
```

Comment:
How about we add a test to make sure `foo.baz` will be rejected?

Reply:
Added a test with the latest commit.